web-app/js/portal/search/FacetedSearchResultsDataView.js

```diff
@@ -61,7 +61,7 @@ Portal.search.FacetedSearchResultsDataView = Ext.extend(Ext.DataView, {
                     return (this.isRecActive(values.uuid)) ? OpenLayers.i18n('collectionExistsMsg') : OpenLayers.i18n("addDataCollectionMsg");
                 },
                 getIconUrl: function(values) {
-                    return Ext.ux.Ajax.constructProxyUrl(this.getGeonetworkImageUrl(values.iconSourceUuid));
```

Comment:
I would keep the proxy. It means we keep everything coming from https.

Reply:
Thanks @pmbohm, I added the imageBucket URL to allowed hosts and returned the use of a proxy. Tested and working from my end :+1: